scripts/run_portserver.py

```diff
@@ -58,7 +58,8 @@
 import threading
 
 from core import utils
-from typing import Callable, Deque, Final, List, Optional, Sequence
+from typing import Callable, Deque, Final, List, Optional, Sequence, cast
+from typing_extensions import Buffer # pylint: disable=import-only-modules
```

Comment:
This seems to be the only use of the new typing_extensions import. I was wondering why we need it and if we could do without it?